cumulus_library/databases/athena.py

```diff
@@ -105,6 +105,8 @@ def col_pyarrow_types_from_sql(self, columns: list[tuple]) -> list:
                     output.append((column[0], pyarrow.int64()))
                 case "double":
                     output.append((column[0], pyarrow.float64()))
+                case "decimal":
+                    output.append((column[0], pyarrow.decimal128(column[4], column[5])))
```

Comment:
I forgot to mention, but duckdb does not need a similar change - the type comes through as NUMBER there, which is already handled.

Reply:
i want a library to just handle 'python types to sql types for this database'